src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -535,34 +535,49 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return MakeEndOfLineTrivia(GetNextChar)
         End Function
 
+        Private Function TryGet(ByRef Ch As Char) As Boolean
+            Dim ok = CanGet()
+            Ch = If(ok, Peek(), Ch)
+            Return ok
+        End Function
+
+        Private Function TryGet(num As Integer, ByRef Ch As Char) As Boolean
+            Dim ok = CanGet(num)
+            Ch = If(ok, Peek(num), Ch)
+            Return ok
+        End Function
+
         Private Function ScanLineContinuation(tList As SyntaxListBuilder) As Boolean
-            If Not CanGet() Then
+            Dim ch As Char = ChrW(0)
+            If Not TryGet(ch) Then
                 Return False
             End If
 
             If Not IsAfterWhitespace() Then
                 Return False
             End If
 
-            Dim ch As Char = Peek()
             If Not IsUnderscore(ch) Then
                 Return False
             End If
 
             Dim Here = 1
-            While CanGet(Here)
-                ch = Peek(Here)
-                If IsWhitespace(ch) Then
-                    Here += 1
-                Else
-                    Exit While
-                End If
-            End While
+            Dim skipWhitespaces = TryGet(Here, ch) AndAlso (IsWhitespace(ch) OrElse IsSingleQuote(ch))
```

Comment:
>skipWhitespaces [](start = 16, length = 15)

This can be simplified (always skip whitespaces):
```vb
                Dim here = 1
                While TryGet(Here, ch)
                    If IsWhitespace(ch) Then
                        Here += 1
                    Else
                        Exit While
                    End If
                End While
```

Reply:
I also thought we could do that, but wasn't 100% sure.

Reply:
@AdamSpeight2008 I don't even know GetWhiteSpaceLenght existed.